quay-enterprise/aci-signing-keys.md

```diff
@@ -0,0 +1,45 @@
+# Generating signing keys for ACI conversion
+
+This document explains how to add a pair of signing keys to your Quay Enterprise installation allowing Quay to sign container images after converting them to ACI format.
+
+## Download the generation script and config
+
+Download the files [aci-signing-key-batch](aci-signing-key-batch) and [generate-signing-keys.sh](generate-signing-keys.sh) next to your configuration directory.
+
+## Make sure the generation script is executable
+
+```sh
+chmod +x generate-signing-keys.sh
+```
+
+## Edit the configuration
+
+Edit the [aci-signing-key-batch](aci-signing-key-batch) configuration, replacing the email address, name and comment.
+
+## Run the generation script
+
+Run the generation script, pointing it to an output config directory.
```

Comment:
Run the generate script, giving the name of an output directory as the argument: